bin/docserv-createconfig

```diff
@@ -27,40 +27,68 @@ import re
 
-def generate_config(languages, branches, version="0.9.9", contact=None):
+
+def prettify(tree, indent="  "):
+    """Return a pretty-printed XML string for the Element.
+
+    :param tree: The XML tree
+    :type tree: :class:`xml.etree.ElementTree.ElementTree`
+
+    :param indent: string used for indentation (usually spaces)
+    :type indent: str
+
+    :return: the indented string
+    :rtype: str | byte
+    """
+    from xml.dom import minidom
+    from xml.etree import ElementTree
+    rough_string = ElementTree.tostring(tree.getroot(), encoding='UTF-8')
+    reparsed = minidom.parseString(rough_string)
+    return reparsed.toprettyxml(indent=indent)
+
+
+def generate_config(languages, branches, version="0.9.10", contact=None):
     """Generate the configuration file.
 
     :param language: The allowed languages
     :type languages: list
 
-    :param branches:
-    :type branches:
+    :param branches: The branch name and their DC files, for example:
+               {'develop': ['DC-FOO-all', 'DC-FOO-install-quick'], ... }
+    :type branches: dict
 
     :param str schemaversion: the schema version to use
     :param str contact: the contact email address
     """
-    root = etree.Element("product", productid="FIXME", schemaversion=version)
-    etree.SubElement(root, "name").text = "FIXME"
-    etree.SubElement(root, "shortname").text = "FIXME"
+
+    root = etree.Element("product", productid=FIXME, schemaversion=version)
+    etree.SubElement(root, "git", remote=GIT_REMOTE_URL)
+    etree.SubElement(root, "name").text = FIXME
+    etree.SubElement(root, "shortname").text = FIXME
     maintainers = etree.SubElement(root, "maintainers")
-    etree.SubElement(maintainers, "contact").text = "FIXME" if not contact else contact
+    etree.SubElement(maintainers, "contact").text = FIXME if not contact else contact
 
     for language in languages:
-        description = etree.SubElement(root, "desc", default="0", lang=language)
-        etree.SubElement(description, "p").text = "FIXME"
+        description = etree.SubElement(root, "desc",
+                                       default="0" if language != 'en-us' else "1", lang=language)
+        etree.SubElement(description, "p").text = FIXME
 
     for branch in branches:
         if branch.startswith("trans/"):
             continue
-        docset = etree.SubElement(root, "docset", setid=branch.replace("maintenance/", ""), lifecycle="FIXME")
-        etree.SubElement(docset, "version").text = "FIXME"
+        docset = etree.SubElement(root, "docset", setid=branch.replace("maintenance/", ""), lifecycle=FIXME)
+        etree.SubElement(docset, "version").text = FIXME
         builddocs = etree.SubElement(docset, "builddocs")
-        git = etree.SubElement(builddocs, "git")
-        etree.SubElement(git, "remote").text = "FIXME"
+        # Optionally <git>, look at product/git
+        # etree.SubElement(builddocs, "git", remote=GIT_REMOTE_URL)
```

Comment:
Not true anymore, since the `<git>` positional change was reverted. Also, `remote` should now be an attribute.

Reply:
True. Also something for #117